Gopkg.lock

```diff
@@ -3,103 +3,67 @@
 
 [[projects]]
   branch = "master"
```

Comment:
If the changes to this file are due to a newer version of dep, let's keep them.
Else let's ditch them.

Reply:
The file is removed! (I was running an old version of `dep`)